README.md

```diff
@@ -217,8 +239,9 @@ Firstly thanks!  We value your time and will do our best to review the PR as soo
 4. Run the tests by `$ make test`
 5. Setup environment variable `TF_ACC` to enable online acceptance test cases by `$ export TF_ACC=1`
 6. Setup environment variables `IC_USERNAME` and `IC_API_KEY` of your provisioning API to grant online acceptance test cases permission by `$ export IC_USERNAME=<your instaclustr username>` and `$ export IC_API_KEY=<your provisioning API key>`
-7. Run the acceptance tests `$ make testacc`
-8. Create a PR and send it our way :)
+7. Setup environment variable `KMS_ALIAS` and `KMS_ARN` to your AWS KMS key values to run test cases for EBS encryption of nodes by `$ export KMS_ALIAS=<your KMS alias>` and `$ export KMS_ARN=<your KMS ARN>`
```

Comment:
Could a note be added here that the tester needs to make sure th ARN cannot have been already added since duplicates are not allowed?